db/seeds.rb

```diff
@@ -190,6 +190,7 @@
     username: "Admin_McAdmin",
     profile_image: File.open(Rails.root.join("app/assets/images/#{rand(1..40)}.png")),
     confirmed_at: Time.current,
+    registered_at: Time.current,
```

Comment:
great catch!

This only happens for the admin user, since the other users are generated via the [factory](https://github.com/forem/forem/blob/main/spec/factories/users.rb#L24) and we set registered at for those users.